templates/workflows-kms-key.yaml

```diff
@@ -0,0 +1,35 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Description: KMS key for workflows infrastructure
+
+Parameters:
+  TemplateRootUrl:
+    Type: String
+    Description: URL of S3 bucket where templates are deployed
+    ConstraintDescription: Must be a valid S3 HTTP URL
+
+Resources:
+  WorkflowsKeyStack:
```

Comment:
@zaro0508 The design I have chosen allows me to create the template once in common. Otherwise, to use kms-key.yaml directly, I have to create the template twice in dev and prod. Unless there's some clever sceptre functionality I've overlooked?

Reply:
it is inadequate; see below